sig-network/annual-report-2023.md

```diff
@@ -4,6 +4,27 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
+  - Governance and leadership changes
+
+    - Casey Davenport (**[@caseydavenport](https://github.com/caseydavenport)**)
+      and Dan Williams (**[@dcbw](https://github.com/dcbw)**) stepped
+      down as Chairs, Shane Utt (**[@shaneutt](https://github.com/shaneutt)**) and
+      Mike Zappa (**[@mikezappa87](https://github.com/mikezappa87)**) became new
+      Chairs (joining Tim Hockin (**[@thockin](https://github.com/thockin)**), who
+      remained a Chair)
+
+    - Dan Winship (**[@danwinship](https://github.com/danwinship)**) and
+      Antonio Ojea (**[@aojea](https://github.com/aojea)**) became SIG Network's
+      first Tech Leads.
+
+  - Major KEPs? FIXME
+
+  - Gateway API
+
+    [Gateway API is now GA!](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/)
+
+  - NetworkPolicy WG? FIXME
```

Comment:
The Network Policy API subgroup to SIG network had a great 2023.  The community mainly focused on the maintenance and new feature development regarding our two major APIs [AdminNetworkPolicy](https://network-policy-api.sigs.k8s.io/api-overview/#the-adminnetworkpolicy-resource) and [BaselineAdminNetworkPolicy](https://network-policy-api.sigs.k8s.io/api-overview/#the-baselineadminnetworkpolicy-resource).  This  also included a few talks:

- [Sig-Network-Policy-API Intro and Project Update: Kubecon NA 2023](https://youtu.be/lYWW3KogPTg?si=c6vpOwWMp5D6GpMM)
- [AdminNetworkPolicy: Kubecon NA 2023](https://youtu.be/DTxvTCISi7Q?si=pz8RN35ptn8eFeE5)

Additionally major features such as [Egress Traffic Control](https://network-policy-api.sigs.k8s.io/npeps/npep-126-egress-traffic-control/), [FQDN Selectors](https://network-policy-api.sigs.k8s.io/npeps/npep-133/) and [Network Tenancy](https://network-policy-api.sigs.k8s.io/npeps/npep-122/) all made great progress during the year and we are excited to deliver them here in 2024. 

 On top of the APIs and Features, we introduced the [Policy-Assistant](https://github.com/kubernetes-sigs/network-policy-api/tree/main/cmd/policy-assistant) tool which allows users to have a better overview of how all of the native Kubernetes APIs interact on real clusters.

As always we continue to ask the broader community to get involved, weather that's trying out our APIs, updating docs, or helping our with NPEP design we are hugely appreciative of all help. Please checkout [our website](https://network-policy-api.sigs.k8s.io/) for even more information.